packages/gatsby/package.json

```diff
@@ -147,8 +147,10 @@
   },
   "files": [
     "./cache-dir",
-    "./dist"
+    "./dist",
+    "./index.d.ts"
```

Comment:
Does adding this have any effect on people that aren't using TypeScript?

Reply:
IIRC, shouldn't be a problem. People who use e.g. Flow should still get their Flow typechecking results in their IDE.